charts/api/values.yaml

```diff
@@ -117,6 +117,9 @@ app:
   jwtSecretSecretKey: SomeSecretKey
   queryServiceHost: someHost:9999
   elasticSearchHost: someHost:1234
+  # elasticSearchSharedIndexHost: somehost:1234
```

Comment:
If the desired behavior is to not include the env vars derived from these keys, the common thing to do is to `null` them out. Like this, it's kind of hard to say / confusing whether this is commented out on purpose or not.